rabbus.go

```diff
@@ -303,3 +311,16 @@ func notifyClose(dsn string, r *rabbus) {
 		}
 	}
 }
+
+func declareExchange(ch *amqp.Channel, ex, kind string, config Config) error {
```

Comment:
It is fine for now, just as a info, hope soon I will declared this method to rabbus struct which will be renamed to smith else, so in the constructor wi'll have:
```go
type Rabbus interface {
 // methods omitted
}

func NewRabbus(c Config) (*RabbusAmqp, error) {
return &RabbusAmqp{}
}
```

This way we can declare private methods on the struct freely without the compiler complaining about method is not declared